troubleshooting/debugging.md

```diff
@@ -608,6 +609,23 @@ In other cases, the original config would be correct. For example, assume your a
 
 ### Fluent Bit Windows containers
 
+#### Enabling debug mode for Fluent Bit Windows images
+Debug mode can be enabled for Fluent Bit Windows images so that when Fluent Bit crashes, we are able to generate crash dumps which would help in debugging the issue. This can be accomplished by overriding the entrypoint command to include the `-EnableCoreDump` flag. For example-
+```
+# If you are using the config file located at C:\fluent-bit\etc\fluent-bit.conf 
+Powershell -Command C:\\entrypoint.ps1 -EnableCoreDump
```

Comment:
what is `C:\\entrypoint.ps1` - where do I find it? 

I don't see it top level in our repo like the linux entrypoint. Sorry this is probably something I should know which you can help me learn now.

Reply:
This script is present in our repository at [this location](https://github.com/aws/aws-for-fluent-bit/blob/mainline/scripts/entrypoint.ps1). 

We are [copying the same](https://github.com/aws/aws-for-fluent-bit/blob/7dba31fa1015966ae0749c9811046fc6b9f69831/Dockerfile.windows#L22) into the root of our container image.

Reply:
Ah thanks